common/beerocks/tlvf/AutoGenerated/src/beerocks_message.cpp

```diff
@@ -54,11 +54,9 @@ message::sUdsHeader *message_com::get_uds_header(ieee1905_1::CmduMessage &cmdu)
 bool message_com::intel_oui(
     std::shared_ptr<ieee1905_1::tlvVendorSpecific> tlv_vendor_specific_header)
 {
-    // FIXME: https://github.com/prplfoundation/prplMesh/issues/33
-    // return ieee1905_1::tlvVendorSpecific::eVendorOUI(
-    //            uint32_t(std::get<1>(tlv_vendor_specific_header->vendor_oui(0))) & 0x00FFFFFF) ==
-    //        ieee1905_1::tlvVendorSpecific::eVendorOUI::OUI_INTEL;
-    return true;
+    return ieee1905_1::tlvVendorSpecific::eVendorOUI(
+                uint32_t(tlv_vendor_specific_header->vendor_oui()) && 0x00FFFFFF) ==
```

Comment:
`&&` can't be right. But if the masking moves into the vendor_oui function, this goes away.

Reply:
Yup, already addressed by Semyon. This is going to go away.